TokenMenu.js

```diff
@@ -1731,10 +1733,49 @@ function open_quick_roll_menu(e){
 		$("#qrm_dialog").remove();
 	});
 	qrm_title_bar_popout.click(function() {
-		qrm.hide();
+		$("#qrm_dialog").hide();
 		let name = "Quick Roll Menu";
+		//destory the iconselectmenu, since it won't work in the popout
+		$("#qrm_dialog #quick_roll_footer select#qrm_save_dropdown").iconselectmenu( "destroy" );

```

Comment:
This breaks the popout select menus on my end.

try putting this in qrm_update_popout instead

```
		removeFromPopoutWindow("Quick Roll Menu", "#qrm_save_dropdown-button");
		removeFromPopoutWindow("Quick Roll Menu", "#qrm_apply_conditions-button")
		$(childWindows['Quick Roll Menu'].document).find(".general_input").css('display', '');
		```
		
You don't need to reintialize it below in this function either this way.



I think adding something like this to the popout click here would be good too. Otherwise the main window is just hidden and when tokens are added again they actually get removed and a blank window gets openned or only some tokens look like they were added.

```
		$(window.childWindows[name]).on('unload', function(){
			$("#qrm_clear_all").click();
			$("#qrm_dialog").remove();
			$(window.childWindows[name]).off();
		});
```

Reply:
Weird, I didn't have that issue on my end. But its certainly better to just delete it from the popout rather than reinit. 

Yeah, thats probably a good idea, since its a add/remove toggle it and its technically still open but just hidden it can give that result, which is probably not desirable. 

Thanks, I'll add both those changes when I get the chance.